### ISSUE TITLE
Can we remove text on the button?

### ISSUE THREAD

**Issue by banned99** [alexandrtovmach/react-microsoft-login#11]
Hi Development Team,

I really appreciated that you guys implementing this for us.

But can we just include only Microsoft logo? Like a Google button below, or can we custom it?

![image](https://user-images.githubusercontent.com/8780911/57367854-fed57f00-71b3-11e9-8b03-9e4981b6b264.png)

Thanks

**Comment by alexandrtovmach** [comment]
Hello,

This package based on (official Microsoft design)[https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-add-branding-in-azure-ad-apps], and I wouldn't change it to using bad practices, so my answer - no, here's no way to use custom button.

**Comment by banned99** [comment]
Ok thanks anyway :)

**Comment by alexandrtovmach** [comment]
@banned99 

Available in [1.3.0 version](https://www.npmjs.com/package/react-microsoft-login/v/1.3.0)

```
<MicrosoftLogin
  clientId={clientId}
  authCallback={this.loginHandler}
>
  <button className="your-class">Custom button</button>
</MicrosoftLogin>
```
